testing/e2e.sh

```diff
@@ -59,7 +59,7 @@ EOF
   EXPECT_CONTAINS "$(cat output.txt)" "Second: 5"
   EXPECT_CONTAINS "$(cat output.txt)" "Third: 6"
   EXPECT_CONTAINS "$(cat output.txt)" "Fourth: 7"
-  EXPECT_CONTAINS "$(cat bazel-bin/testdata/py3_image_with_custom_run_flags)" "-i --rm --network=host -e ABC=ABC"
+  EXPECT_CONTAINS "$(cat bazel-bin/testdata/py3_image_with_custom_run_flags.executable)" "-i --rm --network=host -e ABC=ABC"
```

Comment:
Does this fail w/o the change?
i.e., this should not be necessary, as the default executable should still have these same contents.

Reply:
Yes, it still seems to fail.  I think because `.executable` has become the new default executable?

Reply:
so what are the contents of `bazel-bin/testdata/py3_image_with_custom_run_flags` after this change?
What happens if you run `bazel run //testdata:py3_image_with_custom_run_flags`? Does it work as expected or does it fail?